dashboard/app/models/course_offering.rb

```diff
@@ -385,6 +386,15 @@ def self.properties_from_file(content)
   # seed_all
   def self.seed_record(file_path)
     properties = properties_from_file(File.read(file_path))
+
+    key = properties[:self_paced_pl_course_offering_key]
```

Comment:
just noting that this is safe (won't overwrite the `key` field on this model) because this is a class method rather than an instance method 👍